### PR TITLE
added instructions for changing --system-site-packages in existing envir...

### PR DESCRIPTION
**Pull request by drevicko**
...onment

In case someone forgets to use the --system-site-packages flag and wants to change it later, it's currently hard to find out how, so I added instructions here.

### CODE REVIEW

File: docs/userguide.rst [hunk 1]
```diff
@@ -136,6 +136,12 @@ This can be used if you have control over the global site-packages directory,
 and you want to depend on the packages there. If you want isolation from the
 global system, do not use this flag.
 
+If you need to change this setting after creating a virtual environment, you can remove 
+the file ``no-global-site-packages.txt`` from ``lib/python2.7/`` in the environments
+directory. An easy way to get there is with the ``cdsitepackages`` command with the
```
Comment (gaborbernat) [inline docs/userguide.rst]: this ``cds`` seems like a typo, can you please fix it? thanks!

Comment (drevicko) [reply docs/userguide.rst]: It's a command from virtualenvwrapper - I'll add an explanation to that efect

File: docs/userguide.rst [hunk 2]
```diff
@@ -166,6 +166,12 @@ This can be used if you have control over the global site-packages directory,
 and you want to depend on the packages there. If you want isolation from the
 global system, do not use this flag.
 
+If you need to change this option after creating a virtual environment, you can 
+add (to turn off) or remove (to turn on) the file ``no-global-site-packages.txt`` 
+from ``lib/python2.7/`` or equivalent in the environments directory. If you have
```
Comment (gaborbernat) [inline docs/userguide.rst]: I would rather not have mention here of higher-level abstractions/tools. Let's just remove the last sentence and it's good to go (that should be part of ``virtualenvwrapper`` documentation, not here).

Comment (drevicko) [reply docs/userguide.rst]: done..

**Comment by gaborbernat** [comment]
The CI fails because of trailing whitespaces, run tox -e fix_lint to fix that, thanks!

**Pull request merged**